backend/tracim_backend/config.py

```diff
@@ -131,10 +139,79 @@ def get_raw_config(
         )
         return config_value
 
-    # INFO - G.M - 2019-04-05 - Config loading methods
+    # INFO - G.M - 2019-04-05 - load of enabled app
+    def load_enabled_apps(self) -> None:
+        self._load_enabled_apps_config()
+        loaded_apps = load_apps()
+        self._load_enabled_app(self.APP__ENABLED, loaded_apps)
 
-    def load_config(self) -> None:
-        """Parse configuration file and env variables"""
+    def _load_enabled_apps_config(self) -> None:
+        self.log_config_header("App Enabled config parameters:")
+        default_enabled_app = (
+            "contents/thread,"
+            "contents/file,"
+            "contents/html-document,"
+            "contents/folder,"
+            "{extend_apps}"
+            "share_content,"
+            "upload_permission,"
+            "gallery"
+        )
+        extend_apps = ""
+        # TODO - G.M - 2020-01-09 - remove this retrocompat code, as
+        #  CALDAV__ENABLED and COLLABORATIVE_DOCUMENT_EDITION__ACTIVATED
+        # should be deprecated.
+        self.CALDAV__ENABLED = asbool(self.get_raw_config("caldav.enabled", "false"))
```

Comment:
CALDAV__ENABLED and COLLABORATIVE_DOCUMENT_EDITION__ACTIVATED are deprecated. I create a retrocompat code, but i'm not sure that's a good idea.

Reply:
@lebouquetin have you any idea about how to handle these old parameters ?

Reply:
what do you mean by `I create a retrocompat code,`? You did write some code which allow to re-use these parameters? What is the behavior?

Reply:
The good way to fix this:

- default values:
    - agenda actived
    - collabora not activated
- backward compatibility:
    - add a warning if parameters are present (but do not automatically add apps)

Reply:
done.